app/src/main/java/com/adit/tictactoe/MainActivity.java

```diff
@@ -8,21 +8,26 @@
 import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.adit.tictactoe.dialogs.SelectPlayerDialog;
+
 public class MainActivity extends AppCompatActivity {
-     // Player Representation:
+    SelectPlayerDialog dialog;
+    // Player Representation:
     //0 - X
     //1 - O
-    int activePlayer=0;
+    static public int activePlayer=-1;
     boolean gameActive=true;
     int[] gameState={2,2,2,2,2,2,2,2,2};
     // State meanings:
     // 0-X      1-O      2-Null
     int winPositions[][]={{0,1,2},{3,4,5},{6,7,8},
-                          {0,3,6},{1,4,7},{2,5,8},
-                          {0,4,8},{2,4,6}};
+            {0,3,6},{1,4,7},{2,5,8},
```

Comment:
Please maintain the indent that was there initially.

Reply:
Actually what are you seeing is a outdated version it was then changed because when i made a pull request another PR was merged which is for changing array declaration from C style to Java. SO, i had a "merge to upstream resolve" and i changed it to earlier indentation

Reply:
Allthough it was outdated i made sure in recent PR #17 that earlier indentation is maintained